cosmos-admin/README.md

```diff
@@ -1,19 +1,26 @@
 #<a name="top"></a>Cosmos - Cosmos Admin
 
 * [What is cosmos-admin](#whatis)
-* [`data_copier.sh`](#datacopier)
-* [Reporting issues and contact informatio](#contact)
+* [Scripts](#scripts)
+    * [`data_copier.sh`](#datacopier)
+    * [`get_user_stats.sh`](#getuserstats)
+* [OS programming regarding the administrative scripts](#osprogramming)
+    * [Crontab](#crontab)
+    * [Log rotation](#logrotation)
+* [Reporting issues and contact information](#contact)
 
 ##<a name="whatis"></a>What is cosmos-admin
 cosmos-admin is a set of tools designed to administrate a Cosmos deployment, both for the Sahara-based and shared Hadoop-based flavours.
 
 Available tools are:
 
-* **data_copier.sh**: a script designed to copy HDFS data from one cluster to another.
+* **data\_copier.sh**: a script designed to copy HDFS data from one cluster to another.
+* **get\_user\_stats.sh**: a script designed to get certain user statistics (e.g. HDFS use, last access time) in a periodic fashion (i.e. as script within the crontab).
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/5c07eed2af6cdb9c7cb9b61aaf34991335670db2